src/DlibDotNet.Native/dlib/matrix/matrix.h

```diff
@@ -134,6 +134,12 @@ dlib::matrix<__TYPE__, __ROWS__, __COLUMNS__>& m = *(static_cast<dlib::matrix<__
 auto tmp = -m;\
 *ret = new dlib::matrix<__TYPE__, __ROWS__, __COLUMNS__>(tmp);
 
+#define matrix_operator_invert_template(__TYPE__, error, __ELEMENT_TYPE__, __ROWS__, __COLUMNS__, ...) \
+dlib::matrix<__TYPE__, __ROWS__, __COLUMNS__>& m = *(static_cast<dlib::matrix<__TYPE__, __ROWS__, __COLUMNS__>*>(matrix));\
+auto tmp = inv(m);\
```

Comment:
Could you add namespace to inv?
````
dlib::inv
````

### Reference
https://github.com/takuya-takeuchi/DlibDotNet/blob/47436a573c931fc9c9107442b10cf32524805378/src/DlibDotNet.Native/dlib/matrix/matrix_utilities.h#L128

Reply:
Sorry, I forgot to request one more thing.
Could you use develop branch?